### PR TITLE
Some more comments

### PR DESCRIPTION
**Pull request by ineiti**
Added comments in some of the interfaces and classes with regard to
the used time interval formats

### CODE REVIEW

File: dpppt-backend-sdk/dpppt-backend-sdk-data/src/main/java/org/dpppt/backend/sdk/data/DPPPTDataService.java [hunk 1]
```diff
@@ -18,43 +18,43 @@
...
 	void upsertExposees(List<Exposee> exposees, String appSource);
 
 	/**
 	 * Returns the maximum id of the stored exposed entries fo the given batch.
 	 * 
-	 * @param batchReleaseTime
-	 * @param batchLength
-	 * @return
+	 * @param batchReleaseTime in milliseconds since the start of the Unix Epoch, must be a multiple of
+	 * @param batchLength im milliseconds
```
Comment (ubamrein) [inline dpppt-backend-sdk/dpppt-backend-sdk-data/src/main/java/org/dpppt/backend/sdk/data/DPPPTDataService.java:40]: i~m~n typo

File: dpppt-backend-sdk/dpppt-backend-sdk-data/src/main/java/org/dpppt/backend/sdk/data/DPPPTDataService.java [hunk 2]
```diff
@@ -18,43 +18,43 @@
...
 
 	/**
 	 * Returns all exposees for the given batch.
-	 * 
-	 * @param batchReleaseTime
-	 * @param batchLength
-	 * @return
+	 *
+	 * @param batchReleaseTime in milliseconds since the start of the Unix Epoch, must be a multiple of
+	 * @param batchLength im milliseconds
```
Comment (ubamrein) [inline dpppt-backend-sdk/dpppt-backend-sdk-data/src/main/java/org/dpppt/backend/sdk/data/DPPPTDataService.java:49]: i~m~n

File: dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/util/ValidationUtils.java [hunk 3]
```diff
@@ -40,6 +56,12 @@ public boolean isValidBase64Key(String value) {
 		}
 	}
 
+	/**
+	 * Check if the given date is in the range of [now - retentionPeriod ... now], exclusive
```
Comment (ubamrein) [inline dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/util/ValidationUtils.java:60]: it is a closed interval isn't it? so inclusive? or else we should use parenthesis

File: dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/util/ValidationUtils.java [hunk 4]
```diff
@@ -51,15 +73,23 @@ public boolean isDateInRange(OffsetDateTime timestamp) {
...
+	 * @param keyDate in milliseconds since Unix epoch (1970-01-01)
+	 * @return if keyDate represents midnight UTC
 	 */
 	public boolean isValidKeyDate(long keyDate) {
 		return LocalDateTime.ofInstant(Instant.ofEpochMilli(keyDate), ZoneOffset.UTC).toLocalTime().equals(LocalTime.MIDNIGHT);
 	}
 
+	/**
+	 * Check if the given batchReleaseTime is the beginning of a batch, and if it is between
+	 * [now - retentionPeriod ... now], exclusive.
```
Comment (ubamrein) [inline dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/util/ValidationUtils.java:87]: same as above?

File: dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/util/ValidationUtils.java (hunk 3 above)
Comment (ineiti) [reply dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/util/ValidationUtils.java:60]: it's exclusive, so I use `()`. But in fact it's using `Instant.now()` twice, so I think it's off by some milliseconds anyway...

Comment (ineiti) [reply dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/util/ValidationUtils.java]: OK, after discussion on slack, it's `inclusive`... Sorry for the mixup.

**Pull request merged**